reliability/tasks/Task.py

```diff
@@ -101,45 +152,112 @@ def execute(self):
                 if len(all_apps.apps) > 0:
                     apps = list(all_apps.apps.keys())
                     apps_to_build = self.get_targets(apps, self.get_percent())
+                    # prepare parameters list - app name and kubeconfig - for app.build
+                    app_build_args = []
                     for app_to_build in apps_to_build:
-                        all_apps.apps[app_to_build].build()
+                        try:
+                            project = all_apps.apps[app_to_build].project
+                            user = all_projects.projects[project].user
+                            kubeconfig = global_data.kubeconfigs[user]
+                            app_build_args.append((app_to_build, kubeconfig))
+                        except KeyError:
+                            self.logger.info(f"Projet {project_to_modify} no longer exists in projects list.")
```

Comment:
Don't think the project_to_modify variable will be set here, need to change to just _project_ or change the logging line. 
Missing **c** in project as well at the beginning of the line

Reply:
Thank you for the catching!

Reply:
Thank you for the catching!